src/app/settings/shared/settings.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from "@angular/core";
+
+import { AsyncStorageService } from "@/services/storage/async-storage.service";
+
+@Injectable()
```

Comment:
Services must be provided in the root due to lazy loading.

```suggestion
@Injectable({ providedIn: "root" })
```